server/api.go

```diff
@@ -89,7 +103,7 @@ func (p *Plugin) handleUserInstalls(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	installsForUser, err := p.getUpdatedInstallsForUser(req.UserID)
+	installsForUser, err := p.getUpdatedInstallsForUser(req.UserID, false)
```

Comment:
nit: I struggle with the opaque `, false` and `, true` additions. Worth a `getUpdatedInstallsForUserWithSensitive` to be explicit?

Reply:
Thanks for the input! I added new explicit calls for obtaining installs with sensitive info or not.